metadata_backend/api/handlers.py

```diff
@@ -474,9 +474,18 @@ async def submit(self, req: Request) -> Response:
                     LOG.error(reason)
                     raise web.HTTPBadRequest(reason=reason)
                 LOG.debug(f"submission has action {action}")
-                actions[attr["schema"]] = action
+                if action == "release":
```

Comment:
thought we will not do `release`
maybe throwing a 400 - HTTPBadRequest with reason not implemented or 501 whihc is not implemented

Reply:
Honestly, just forgot to remove this part when I was removing part that handled `RELEASE` actions 😅